src/test/suite/fixtures/recorded/languages/rust/chuckNameAir.yml

```diff
@@ -0,0 +1,36 @@
+languageId: rust
+command:
+  spokenForm: chuck name air
+  version: 2
+  targets:
+    - type: primitive
+      mark: {type: decoratedSymbol, symbolColor: default, character: a}
+      modifiers:
+        - type: containingScope
+          scopeType: {type: name}
+  usePrePhraseSnapshot: true
+  action: {name: remove}
+initialState:
+  documentContents: |
+    match user {
+        User { first_name: "John", last_name: last_name } => {}
```

Comment:
Can you change this test so the name binding differs from the field name? It's unclear from this test which one it picked to delete.